charts/redpanda/go.mod

```diff
@@ -0,0 +1,219 @@
+module github.com/redpanda-data/redpanda-operator/charts/redpanda
+
+go 1.23.5
```

Comment:
Our go version is currently `1.23.2`. Where did this come from?

Reply:
from go mod tidy I will try to find exact line